common/src/main/java/org/conscrypt/OpenSSLKey.java

```diff
@@ -313,7 +313,7 @@ static PublicKey getPublicKey(X509EncodedKeySpec keySpec, int type)
     PrivateKey getPrivateKey() throws NoSuchAlgorithmException {
         switch (NativeCrypto.EVP_PKEY_type(ctx)) {
             case NativeConstants.EVP_PKEY_RSA:
-                return new OpenSSLRSAPrivateKey(this);
+                return  OpenSSLRSAPrivateKey.getInstance(this);
```

Comment:
Nit: extra space between `return` and `OpenSSLRSAPrivateKey`.

Reply:
Thanks! Done.